zh/guide/ch_invest.docbook

```diff
@@ -13,327 +13,250 @@
-
-    <para>To setup investment accounts in &app; you can either use the predefined investment account hierarchy
-      or create your own. The minimum you need to do to track investments is to setup an asset
-      account for each type of investment you own. However, as we have seen in previous chapters, it
-      is usually more logical to create a structured account hierarchy, grouping related investments
-      together. For example, you may want to group all your publicly traded stocks under a parent
-      account named after the brokerage firm you used to buy the stocks.
+    <title>创建科目</title>
+
+    <para>要在&app;创建投资科目，可以使用预定义的投资科目结构，也可以自行创建。要跟踪投资，至少需要为每类持有的投资建立一个资产科目。然而，如上文所述，创建结构化的科目层级，并将相关的投资分为一组通常是更有条理的。例如，你可能想要将所有公开交易股票编为一组，放置在以用来买卖股票的经纪公司命名的父科目之下。
     </para>
 
     <note>
-      <para>Regardless of how you setup your account hierarchy, remember that you can always move accounts
-        around later (without losing the work you&rsquo;ve put into them), so your initial account
-        hierarchy does not have to be perfect.
+      <para>请注意，不论怎样组织科目层次，总能将来再随意移动这些科目，同时已经录入的信息不会丢失。所以，一开始建立科目层次无需一步到位。
       </para>
     </note>
 
     <sect2 id="invest_predefined2">
-      <title>Using Predefined Investment Accounts</title>
+      <title>使用预置的投资科目</title>
 
-      <para>The <guilabel>Investment Accounts</guilabel> option of the <guilabel>New Account Hierarchy
-        Setup</guilabel> assistant will automatically create a basic investment account hierarchy
-        for you. To access the predefined investment accounts hierarchy, you must make sure your
-        &app; file is open, switch to the <guilabel>Accounts</guilabel> tab, and choose
+      <para><guilabel>新建科目层次结构</guilabel>向导中的<guilabel>投资科目</guilabel>选项会自动为你创建基本的投资科目层次结构。要使用预置的投资科目层次结构，首先打开&app;文件，切换到<guilabel>科目</guilabel>标签页，选择
         <menuchoice>
-          <guimenu>Actions</guimenu><guimenuitem>New Account Hierarchy</guimenuitem>
+          <guimenu>功能</guimenu><guimenuitem>新建科目层次结构</guimenuitem>
         </menuchoice>
-        . This will run the <guilabel>New Account Hierarchy Setup</guilabel> assistant and allow you
-        to select additional accounts to add to your account hierarchy. Choose the
-        <guilabel>Investment Accounts</guilabel> option (along with any others you are interested
-        in). Assuming only investment accounts were selected, this will create an account hierarchy
-        as shown below.
+        。此菜单期项会启动<guilabel>新建科目层次结构</guilabel>向导，引导你选择要新添加到层次结构中的科目。选择<guilabel>投资科目</guilabel>选项（也可一并选择你感兴趣的其它科目结构）。假设这里只选择了投资科目，则此向导创建的科目结构如下所示。
       </para>
 
       <tip>
-        <para>You can also run the <guilabel>New Account Hierarchy Setup</guilabel> assistant by creating a new
-          &app; file.
+        <para>你也可以通过创建新的&app;文件运行<guilabel>新建科目层次结构</guilabel>向导。
         </para>
       </tip>
 
       <figure>
-        <title>A New File Containinng Only The Default Investment Accounts</title>
+        <title>只包含默认投资科目的新文件</title>
```

Comment:
Yes. Maybe it's better to create screenshots later. Because there's still some work to select better translation for terms (for example how commercial software or textbooks widely used by Chinese accountants express those concepts). GnuCash's expression seems to be organized for US accounting, somewhat. It seems not so interesting to take screenshots each time wanting to change a term, let alone it's even more inconvenience to update GUI translation.

Reply:
I know that problem, too. E.g. English is missing accounting terms, which are common in all other European languages.

And yes most parts of the guide were written by English natives. That results in missing some aspects.
While @CWehli is translating the manual to German, we often have to improve the english version to cover other views.

In general out target is to move from [US-GAAP](https://en.wikipedia.org/wiki/Generally_Accepted_Accounting_Principles_(United_States)) to [IFRS](https://en.wikipedia.org/wiki/International_Financial_Reporting_Standards).

So if the english guide does not cover important aspects,feelfree to send PRs.